ci-operator/config/open-cluster-management/rcm-controller/open-cluster-management-rcm-controller-master.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  base:
+    name: "4.5"
+    namespace: ocp
+    tag: base
+  registration:
+    name: "4.5"
+    namespace: open-cluster-management
+    tag: registration
+  registration-operator:
+    name: "4.5"
+    namespace: open-cluster-management
+    tag: registration-operator
+  registration-operator-src:
+    name: "4.5"
+    namespace: open-cluster-management
+    tag: registration-operator
```

Comment:
i think this needs to be `registration-operator-src`

Reply:
updated